examples/ex9p.cpp

```diff
@@ -426,6 +504,13 @@ FE_Evolution::FE_Evolution(HypreParMatrix &_M, HypreParMatrix &_K,
    M_solver.SetPrintLevel(0);
 }
 
+void FE_Evolution::ImplicitSolve(const double dt, const Vector &x, Vector &k)
+{
+   K.Mult(x, z);
```

Comment:
Please add
z += b;
as in Mult(...).
It adds boundary conditions and sources.

Reply:
Good catch. I had tested with zero BCs and no source. Fixed now.